runtime/src/accounts.rs

```diff
@@ -395,6 +395,36 @@ impl Accounts {
         })
     }
 
+    pub fn load_largest_accounts(
+        &self,
+        ancestors: &Ancestors,
+        num: usize,
+        pubkeys: &[Pubkey],
+        exclude: bool,
```

Comment:
I found `pubkeys`/`exclude` a little confusing.  How about something more like this:
```suggestion
        filter_by_address: &HashSet<Pubkey>,
        filter: AccountAddressFilter,
```
where
```rust
pub enum AccountAddressFilter {
     Exclude,  // exclude all addresses matching the fiter
     Include, // only include addresses matching the filter
}
```
(note the HashSet to avoid a slower linear search of `pubkeys`)

Reply:
Ah, thanks, that's definitely better.